src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -204,12 +204,13 @@
     "BATCH": {
       "CREATE": "course/v1/batch/create",
       "UPDATE": "course/v1/batch/update",
-      "ADD_USERS":"course/v1/admin/enroll", 
+      "ADD_USERS": "course/v1/batch/user/add",
       "GET_DETAILS": "course/v1/batch/read",
       "GET_PARTICIPANT_LIST": "course/v1/batch/participants/list",
       "GET_BATCHS": "course/v1/batch/list",
-      "REMOVE_USERS": "course/v1/admin/unenroll"
-    },
+      "REMOVE_USERS": "course/v1/batch/user/remove",
```

Comment:
@sanjeev-scm @paritshivani can u guys please confirm this change @snehalsabade

Reply:
Not updated by me

Reply:
I did not update